src/components/_cards/CellarDetailsCard.tsx

```diff
@@ -82,7 +86,22 @@ const CellarDetailsCard: VFC<CellarDetailsProps> = ({
 
   // const colors = moveColors(barChartTheme)
 
-  const protocolIcon = protocolsImage[protocols]
+  const isManyProtocols = typeof protocols === "object"
+  const protocolData = isManyProtocols
+    ? protocols.map((v) => {
+        return {
+          title: v,
+          icon: protocolsImage[v],
+        }
+      })
+    : {
+        title: protocols,
+        icon: protocolsImage[protocols],
+      }
```

Comment:
`isManyProtocols` it means its an array right? 
`typeof protocols === object` = `true` = not an array
recheck this block of codes

Reply:
Just update this, using isArray from lodash